app-backup/deja-dup/metadata.xml

```diff
@@ -1,6 +1,10 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <!DOCTYPE pkgmetadata SYSTEM "http://www.gentoo.org/dtd/metadata.dtd">
 <pkgmetadata>
+  <maintainer type="person">
+    <email>petronio@petroniocoelho.com</email>
+    <name>Petronio Coelho</name>
+  </maintainer>
   <maintainer type="project">
```

Comment:
Have you discussed this with GNOME project? If not, you should remove them and add proxy-maint instead.